chromadb/utils/embedding_functions.py

```diff
@@ -402,6 +414,12 @@ def _download(self, url: str, fname: str, chunk_size: int = 1024) -> None:
             for data in resp.iter_content(chunk_size=chunk_size):
                 size = file.write(data)
                 bar.update(size)
+        if not _verify_sha256(fname, self._MODEL_SHA256):
```

Comment:
Instead of error'ing, would it make sense to retry?

Reply:
Makes sense. I'll add `tenacity` retry.

Reply:
Added 3 retries + rethrow.